src/lib/utilities/__mocks__/$app/env.ts

```diff
@@ -0,0 +1,2 @@
+// jsDom works for some dom implementation stuffs but $app/env browser fails whenever it's imported so lets mock it instead
+export const browser = jest.fn(() => false);
```

Comment:
This may be useful in more places, but this allowed me to run store logic and the like in jest.